component-samples/demo/aio/service.yml

```diff
@@ -22,37 +32,36 @@ system-properties:
   
 
 http-server:
-  base_path: app-data
+  base_path: .
   http_port: 8080
   https_port: 8443
   http_schemes:
   - http
   - https
   http_timeout: 20000
   keystore: 
-    path: ssl.p12
-    store-type: PKCS12
-    password: $(ssl_password)
+    path: ./app-data/ssl-host.pem
+    store-type: PEM
+    password: ''
     alias: ssl
-  auth:
-    method: DIGEST
-    role: api
-    user_name: apiUser
-    user_password: $(api_password)
-  subject_names:
-    - DNS:localhost
-    - IP:127.0.0.1
+  truststore: ./app-data/ca-certs.pem
+  certificate_verification: want
```

Comment:
Can you change it to a boolean value?

Reply:
Instead of` want` ?

Reply:
The value is defined by tomcat and not by fdo.   Values can be none, required, or want (per tomcat documentation)